monitor/events/list-get-example-actorsid-resourcesid-error/list-get-example-actorsid-resourcesid-error.2.x.js

```diff
@@ -11,6 +11,6 @@ var params = {
 };
 client.events.list(params, function(error, events) {
     for (var e in events) {
-        process.stdout.write(e.description);
+        process.stdout.write(e);
```

Comment:
What do you mean with `we're not supporting old versions like this`?
The other snippets I deleted were for the old version of the SDKs were the the library didn't allow to do what the snippet described. Those libraries are not maintained anymore so they are not going to include that funcionality.